### PR TITLE
ROC-5069 claim status part admit by set date shows full amount paid

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5069

### Change description

Update the claim status to show defendant amount paid

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

File: src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk [hunk 1]
```diff
@@ -27,9 +27,9 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
-      { defendantName: claim.claimData.defendant.name, totalAmountTillToday: claim.totalAmountTillToday | numeral,
-        moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+  <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk]: this breaks the journey if you don't do a part admit now. I pulled your branch and when I did a paid in full after settlement agreement reached it shows £0. you need to add logic to show the different amount depending on the state of the claim

Comment (rattur) [reply src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk]: Oh dear will take a look asap

File: src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk [hunk 2]
```diff
@@ -27,9 +27,15 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
-    { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral,
-      moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+    {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk]: this won't work if there as no response. It is possible for the claim to not be responded and the claimant to still do paid in full. I would say you should check for if there is an amount in the response if there is not then just show the totalAmountTillToday and do some test scenarios around paid in full from different claim states

Comment (rattur) [reply src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk]: Ok good point

File: src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk [hunk 3]
```diff
@@ -27,9 +27,15 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
-      { defendantName: claim.claimData.defendant.name, totalAmountTillToday: claim.totalAmountTillToday | numeral,
-        moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+    {% if claim.response.amount %}
+        <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
+            { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral,
```
Comment (singh-surjeet) [inline src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk]: Can key/value pair 
```
defendantName: claim.claimData.defendant.name,
moneyReceivedOn: claim.moneyReceivedOn | date
```
be extracted out of if/else block instead repeating them?

**Pull request merged**